pyfolio/deprecate.py

```diff
@@ -0,0 +1,45 @@
+"""Utilities for marking deprecated functions."""
+# Copyright 2016 Quantopian, Inc.
```

Comment:
My non-expert reading of [How to apply this license] (http://choosealicense.com/licenses/apache-2.0/) is that we should include it here:

> The Apache Foundation recommends taking the additional step of adding a boilerplate notice to the header of each source file. You can find the notice at the very end of the license in the appendix.